Spanner/src/Database.php

```diff
@@ -169,6 +169,11 @@ class Database
      */
     private $isRunningTransaction = false;
 
+    /**
+     * @var string
```

Comment:
Could we please document that this could also accept null?

Reply:
Sure, will add in a new PR